install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1166,3 +1166,54 @@ Modify the *_/etc/origin/master/scheduler.json_* file to add the `*kind*` and
 <1> Add `*"kind": "Policy",*`
 <2> Add `*"apiVersion": "v1",*`
 endif::[]
+
+[[manual-upgrades-verifying-the-upgrade]]
+== Verifying the Upgrade
+
+To verify the upgrade, first check that all nodes are marked as *Ready*:
+
+====
+----
+# oc get nodes
+NAME                 LABELS                                                                STATUS
+master.example.com   kubernetes.io/hostname=master.example.com,region=infra,zone=default   Ready
+node1.example.com    kubernetes.io/hostname=node1.example.com,region=primary,zone=east     Ready
+----
+====
+
+Then, verify that you are running the expected versions of the *docker-registry*
+and *router* images, if deployed:
+
+====
+----
+ifdef::openshift-enterprise[]
+# oc get -n default dc/docker-registry -o json | grep \"image\"
+    "image": "openshift3/ose-docker-registry:v3.1.1.6",
+# oc get -n default dc/router -o json | grep \"image\"
+    "image": "openshift3/ose-haproxy-router:v3.1.1.6",
+endif::[]
+ifdef::openshift-origin[]
+# oc get -n default dc/docker-registry -o json | grep \"image\"
+    "image": "openshift/origin-docker-registry:v1.0.6",
+# oc get -n default dc/router -o json | grep \"image\"
+    "image": "openshift/origin-haproxy-router:v1.0.6",
+endif::[]
+----
+====
+
+After upgrading, you can use the experimental diagnostics tool to look for
+common issues:
+
+====
+----
+# openshift ex diagnostics
+...
+[Note] Summary of diagnostics execution:
+[Note] Completed with no errors or warnings seen.
+----
+====
+
+Verify in your old *_/etc/sysconfig/openshift-master_* and
+*_/etc/sysconfig/openshift-node_* files that any custom configuration is added
+to your new *_/etc/sysconfig/atomic-openshift-master_* and
+*_/etc/sysconfig/atomic-openshift-node_* files.
```

Comment:
Regarding the placement of this in general, I think we should end the section w/ the suggestion about running the diagnostics tool (in both this Manual topic and in the equivalent Automated topic spot).